packages/expo-file-system/android/src/main/java/expo/modules/filesystem/ExpokitFilePermissionChecker.java

```diff
@@ -0,0 +1,58 @@
+package expo.modules.filesystem;
+
+import android.content.Context;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.List;
+
+import expo.core.interfaces.InternalModule;
+import main.FilePermissionChecker;
+import main.Permission;
+
+public class ExpokitFilePermissionChecker implements FilePermissionChecker, InternalModule {
```

Comment:
When we add `Interface` suffix to `FilePermissionChecker` interface we can remove the `Expokit` prefix and let it be just `FilePermissionChecker` (or whatever wielder we decide to go with)